src/Container.php

```diff
@@ -1,61 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Phprest;
-
-use function class_exists;
-use function is_null;
-use function sprintf;
-
-class Container extends \League\Container\Container
-{
-    /**
-     * {@inheritDoc}
-     */
-    protected function reflect($class)
-    {
-        // try to reflect on the class so we can build a definition
-        try {
-            $reflection  = new \ReflectionClass($class);
-            $constructor = $reflection->getConstructor();
-        } catch (\ReflectionException $e) {
-            throw new \League\Container\Exception\ReflectionException(
-                sprintf(
-                    'Unable to reflect on the class [%s], does the class exist and is it properly autoloaded?',
-                    $class
-                )
-            );
-        }
-
-        $factory = $this->getDefinitionFactory();
-        $definition = $factory($class, $class, $this);
-
-        if (is_null($constructor)) {
-            return $definition;
-        }
-
-        // loop through dependencies and get aliases/values
-        foreach ($constructor->getParameters() as $param) {
-            $dependency = $param->getType();
-
-            if ($dependency && class_exists($dependency->getName())) {
```

Comment:
Is this if that I added now handled?

Reply:
yes, with  `$errorHandler->treatErrorsAsExceptions(false);` it's not a breaking issue any more.